x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts

```diff
@@ -229,10 +234,44 @@ export const cli = () => {
                   );
                 }
 
+                // Serverless Specific
+                if (vars.serverless) {
+                  log.info(`Serverless mode detected`);
+
+                  if (configFromTestFile?.productTypes) {
+                    vars.kbnTestServer.serverArgs.push(
+                      `--xpack.securitySolutionServerless.productTypes=${JSON.stringify([
+                        ...configFromTestFile.productTypes,
+                        // Why spread it twice?
+                        // The `serverless.security.yml` file by default includes two product types as of this change.
+                        // Because it's an array, we need to ensure that existing values are "removed" and the ones
+                        // defined here are added. To do that, we duplicate the `productTypes` passed so that all array
+                        // elements in that YAML file are updated. The Security serverless plugin has code in place to
+                        // dedupe.
+                        ...configFromTestFile.productTypes,
```

Comment:
I think this PR https://github.com/elastic/kibana/pull/161884 fixed the problem. Probably we can just assign directly now.

Reply:
I'll check again, but I'm thinking that PR did not address when options are passed in via cli (`--xpack.securitySolutionServerless.productTypes=value`).

Reply:
FYI: I validated that its still happening for options passed in via CLI.I have reached out to the Kibana Core team for feedback.

Reply:
you can filter out the values like here 
https://github.com/elastic/kibana/pull/162281/files#diff-9f40ced6d29c4fc2709af881680400293d8ce1bc9ebb07b9138d6d99c83c09c9R194-R202

Reply:
Good catch, it seems they'll do it here https://github.com/elastic/kibana/issues/162842. Thanks for reporting it 👌